edb/graphql/extension.pyx

```diff
@@ -125,6 +125,13 @@ async def handle_request(
                         raise TypeError(
                             '"globals" must be a JSON object')
 
+        elif request.method == b'OPTIONS':
+            response.status = http.HTTPStatus.NO_CONTENT
+            response.custom_headers['Access-Control-Allow-Methods'] = \
+                'POST, GET, OPTIONS'
+            response.custom_headers['Access-Control-Max-Age'] = '86400'
+            return
+
```

Comment:
If we're hiding GraphQL from UI (see https://github.com/edgedb/edgedb-studio/pull/17#issuecomment-1158491258) this change is no longer needed?